types/wordpress__components/package.json

```diff
@@ -2,6 +2,7 @@
     "private": true,
     "dependencies": {
         "@wordpress/element": "^2.14.0",
-        "re-resizable": "^4.7.1"
+        "re-resizable": "^4.7.1",
+        "react": "^16.13.1"
```

Comment:
I think you need to add a reference in the tsconfig here so that react is resolved to the types:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b5adb9d5aba5ec1a54f3a317f77b050bdb61fe24/types/wordpress__components/tsconfig.json#L20-L33

Alternatively, if we only need `csstype` should we pull that in?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4aa885a579ba9313db4d94cdcd258face53ffe99/types/react/package.json#L4

Reply:
I noticed the error as well... but the error says that path mapping is only necessary for namespaced (which makes sense, the dependency name wouldn't match the type directory).

`react` isn't an allowed dependency, so maybe pulling in `csstype` will resolve this like you suggested.